WooCommerce/Classes/ViewRelated/Custom Fields/CustomFieldEditorView.swift

```diff
@@ -0,0 +1,208 @@
+
+                    Button(action: {
+                        showingActionSheet = true
+                    }, label: {
+                        Image(systemName: "ellipsis")
+                            .renderingMode(.template)
+                    })
+                    .confirmationDialog("More Options", isPresented: $showingActionSheet) {
+                        actionSheetContent
+                    }
+                }
+            }
+        }
+        .sheet(isPresented: $showRichTextEditor) {
+            RichTextEditor(html: $value)
+                .onDisappear {
+                    checkForModifications()
+                }
+        }
+    }
+
+    @ViewBuilder
+    private var actionSheetContent: some View {
+        Button("Copy Key") { // todo-13493: set String to be translatable
+            UIPasteboard.general.string = key
+            // todo-13493: Show a notice that the key was copied
+        }
+
+        Button("Copy Value") { // todo-13493: set String to be translatable
+            UIPasteboard.general.string = value
+            // todo-13493: Show a notice that the value was copied
+        }
+
+        Button("Delete Custom Field", role: .destructive) { // todo-13493: set String to be translatable
+            // todo-13493: Implement delete action
+        }
+    }
+
+    private func checkForModifications() {
+        isModified = key != initialKey || value != initialValue
+    }
+
+    private func saveChanges() {
+        // todo-13493: add save logic
+    }
+}
+
+private struct RichTextEditor: View {
+    @Binding var html: String
+    @State private var isModified: Bool = false
+    @Environment(\.presentationMode) var presentationMode
+
+    var body: some View {
+        NavigationView {
+            AztecEditorView(html: $html)
+                .toolbar {
+                    ToolbarItem(placement: .navigationBarLeading) {
+                        Button {
+                            presentationMode.wrappedValue.dismiss()
+                        } label: {
+                            Text("Cancel") // todo-13493: set String to be translatable
+                        }
+                    }
+                    ToolbarItem(placement: .navigationBarTrailing) {
+                        Button {
+                            // todo-13493: implement save action
+                            presentationMode.wrappedValue.dismiss()
+                        } label: {
+                            Text("Done") // todo-13493: set String to be translatable
+                        }
+                        .disabled(!isModified)
+                    }
+                }
+        }
+    }
+}
+
+// MARK: Constants
+private extension CustomFieldEditorView {
+    enum Layout {
+        static let sectionSpacing: CGFloat = 16
+        static let subSectionsSpacing: CGFloat = 8
+        static let cornerRadius: CGFloat = 8
+        static let inputInsets = EdgeInsets(top: 8, leading: 5, bottom: 8, trailing: 5)
```

Comment:
This is the first time I've seen `5` used as padding in the app. Are we sure it shouldn't have been 6 or 8?

Reply:
For more context on this, I was looking at existing UI pattern in the app that shows more than 1 input fields on one screen (key/value input in our case).

That design is surprisingly rare in our app, and I was able to find it in the **Contact Support** screen:
![Simulator Screenshot - iPhone 15 Pro - 2024-09-11 at 10 53 07](https://github.com/user-attachments/assets/4efe1321-72c9-4332-943f-3a74e2db3e76)

The layout here is pretty much a copy from there, here:

https://github.com/woocommerce/woocommerce-ios/blob/a7923ca8ecc7e14ee2ec59882ee147c3c5d91aac/WooCommerce/Classes/ViewRelated/Dashboard/Settings/Help/SupportForm/SupportForm.swift#L264

Reply:
Got it :D I think this is an error also there probably